src/app/utils/route-utils.ts

```diff
@@ -6,14 +6,23 @@ import { EvmTokenType, Layer } from '../../oasis-indexer/api'
 import { Network } from '../../types/network'
 import { SearchScope } from '../../types/searchScope'
 
-export abstract class RouteUtils {
-  private static ENABLED_LAYERS_FOR_NETWORK: Partial<Record<Network, Layer[]>> = {
-    [Network.mainnet]: [Layer.emerald, Layer.sapphire],
-    [Network.testnet]: [
-      // Layer.emerald,
-      Layer.sapphire,
-    ],
+export type SpecifiedPerEnabledLayer<T = any> = {
+  [N in keyof (typeof RouteUtils)['ENABLED_LAYERS_FOR_NETWORK']]: {
+    [L in keyof (typeof RouteUtils)['ENABLED_LAYERS_FOR_NETWORK'][N]]: T
   }
+}
+
+export abstract class RouteUtils {
+  private static ENABLED_LAYERS_FOR_NETWORK = {
+    [Network.mainnet]: {
+      [Layer.emerald]: true,
```

Comment:
Technically, this should be disabled for now, too. (So commented out.)
(It was an oversight that I have accidentally re-enabled it at one point, by leaving in a debug change while merging.)

Reply:
Okay ill try to find a sapphire verified contract example first

Reply:
I have not found it :shrug: